sql/qaqc/qaqc_app.sql

```diff
@@ -0,0 +1,61 @@
+DROP TABLE IF EXISTS APP_qaqc;
```

Comment:
I like this more personally. The only thing I would change is switch the order so `qaqc_APP`, that way all the qaqc tables are next to each other alphabetically which makes them easier to find

Reply:
That would be nice but already all of the other qaqc tables are named the opposite way - can still do it with the plan to change the other ones also down the line